test/unit/terraform-plan-parser.test.ts

```diff
@@ -11,14 +11,20 @@ function readExpected (dataFile: string): ParseResult {
   return dataObj as ParseResult;
 }
 
-async function readActual (dataFile: string): Promise<ParseResult> {
+async function readActual (dataFile: string, clean: boolean): Promise<ParseResult> {
   const stdout = await readFileAsync(path.join(__dirname, 'data', dataFile + '.stdout.txt'),
     { encoding: 'utf8' });
-  return parseStdout(stdout);
+  return parseStdout(stdout, clean);
+}
+
+async function runCleanTest (dataName: string, t: any) {
+  const actual = await readActual(dataName, true);
+  const expected = readExpected(dataName);
+  t.deepEqual(actual, expected);
 }
 
 async function runTest (dataName: string, t: any) {
-  const actual = await readActual(dataName);
+  const actual = await readActual(dataName, false);
```

Comment:
Small improvement, but how would feel about simply adding the `clean: boolean` parameter to `runTest` and pass it through as needed. That would avoid the need for a separate `runCleanTest` function which duplicates most of its logic.

Reply:
That sounds good to me, I actually debated doing it that way when I started looking at testing. I will give the other reviewers a chance to take a peak and then update my change based on all the feedback. :smile: